labs/3-Add-Identity/README.md

```diff
@@ -37,12 +37,21 @@ You can read more about [selecting an identity management solution for ASP.NET C
     ![The dashboard showing the logs for the exited 'idp' resource](./img/dashboard-idp-logs-error.png)
 
     The logs should indicate that there was an error with the imported client `webapp`, specifically that the URLs/URIs configured are invalid. This is because the `eshop-realm.json` file that was imported contains processing tokens intended to inject values from environment variables which haven't been configured yet.
-1.  We can use Aspire APIs to extract the runtime-assigned URLs for our `webapp` resource and inject them into the `idp` resource as environment variables using the [`WithEnvironment` method](https://learn.microsoft.com/dotnet/api/aspire.hosting.resourcebuilderextensions.withenvironment?view=dotnet-aspire-8.0), so that the processing tokens in the imported `eshop-realm.json` file will be replaced with valid values. Add the following lines to the `Program.cs` file, after the call defining the `webapp` resource. You will need to modify the `webapp` resource code to capture the resource in a variable named `webApp`:
+1. We can use Aspire APIs to extract the runtime-assigned URLs for our `webapp` resource and inject them into the `idp` resource as environment variables using the [`WithEnvironment` method](https://learn.microsoft.com/dotnet/api/aspire.hosting.resourcebuilderextensions.withenvironment?view=dotnet-aspire-8.0), so that the processing tokens in the imported `eshop-realm.json` file will be replaced with valid values. Add the following lines to the `Program.cs` file, after the call defining the `webapp` resource. You will need to modify the `webapp` resource code to capture the resource in a variable named `webApp`:
 
     ```csharp
+    // Force HTTPS profile for web app (required for OIDC operations)
+    var webApp = builder.AddProject<WebApp>("webapp", launchProfileName: "https")
```

Comment:
I don't think the launch profile thing is required anymore. I thought I removed it after I got tests working in HTTPS on CI. Are you seeing that it's still required somewhere?

Reply:
You might want to compare with what's in the root ./src solution now:
https://github.com/dotnet-presentations/eshop-app-workshop/blob/66226ea402e0d2cb60c52a3d893414e2947525b3/src/eShop.AppHost/Program.cs#L42-L64

Reply:
Fixed to match end solution for this section